MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.rst LICENSE NOTICE HISTORY.rst requests/cacert.pem
+recursive-include tests *.py
```

Comment:
If we did a release with dirty directories, you're right, that could be a problem. I'd suggest using something like `global-exclude *.py[co]` to solve the `.pyc` issue. That approach will be more inline with most of the other major python projects.